system_test.sh

```diff
@@ -22,7 +22,19 @@ function try { let tests+=1; this="$1"; }
 
 function assert {
 	[[ "$1" == "$2" ]] && { printf "."; return; }
-	printf "\nFAIL: $this\n'$1' != '$2'\n"; exit 1
+	printf "\nFAIL: $this\n'$1' != '$2'\n"
+
+	echo "*************************************"
+	echo "All files in temporary directory and and a peek of their contents:"
+	head -n 50 $(find $tmp -type f)
+	echo "*************************************"
+	echo "Preceding stdout:"
+	cat $tmp/exec.out
+	echo "*************************************"
+	echo "Preceding stderr:"
+  cat $tmp/exec.err
+	echo "*************************************"
```

Comment:
This is becoming simple: lines 31-36 are not redundant because the `find` command also prints the contents of .err and .out files. Let's delete those and this will be good!

Reply:
ok